boards/arm/mikroe_clicker_2/pinmux.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2020 Trifork
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <sys/sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* Pin assignments for MikroE Clicker 2 board for STM32 */
+static const struct pin_config pinconf[] = {
+};
+
+static int pinmux_mikroe_clicker_2_init(const struct device *port)
+{
```

Comment:
This file could be removed now

Reply:
Ok, I thought I read that the idea was to keep it in, but ok.